tests/unit/lms/views/api/gateway_test.py

```diff
@@ -85,6 +180,17 @@ def validator(self, schema):
         return Draft202012Validator(schema)
 
 
+@pytest.fixture
+def assignment():
+    # Use baked values, so we can have static comparisons in the tests
+    return factories.Assignment.create(
+        document_url="document_url",
+        resource_link_id="resource_link_id",
+        title="title",
+        description="description",
+    )
```

Comment:
I'm planning to keep adding more and more joined things and relationships to this assignment as we get more data to dump.